content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,79 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing ways to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+Note that to start using a GDrive remote, you only need to add it with a valid
+URL (see next section), and use any DVC command that requires it (see
+`dvc pull`, `dvc fetch`, `dvc push`). For example:
+
+```dvc
+$ dvc remote add -d myremote gdrive://root/dvc/dvcstore
+$ dvc add file.dat
+$ dvc push -r myremote
+...
+Go to the following link in your browser:
+
+    https://accounts.google.com/o/oauth2/auth # ... copy this link
+
+Enter verification code: # <- enter resulting token
+```
+
+See the [Authorization](#authorization) section for more details.
+
+## URL format for Google Drive remotes
+
+A GDrive remote URLs is needed for the `dvc remote add` command. It can be
+constructed with a _base_, and an **optional** _path_ to a folder i.e.
+`gdrive://<base>/path/to/folder`. The base can be one of:
+
+1. `root` - indicates your topmost Google Drive directory.
+
+```dvc
+$ dvc remote add -d mygdroot gdrive://root
+$ dvc remote add mygdfolder gdrive://root/path/to/folder
+```
+
+2. Google Drive _Folder ID_
+
+To obtain the unique _Folder ID_ in question, navigate into that folder in your
+web browser, and find it in the address bar. For example, for
+`https://drive.google.com/drive/u/0/folders/0AIac4JZqHhKmUk9PDA`:
+
+```dvc
+$ dvc remote add mygdfolder gdrive://0AIac4JZqHhKmUk9PDA
+$ dvc remote add mygdsubfolder \
+                 gdrive://0AIac4JZqHhKmUk9PDA/sub/path
+```
+
+Note that [shared folders](https://support.google.com/drive/answer/7166529) and
+[shared drives](https://support.google.com/a/users/answer/9310351) can only be
+referenced by _Folder ID_.
+
+3. `appDataFolder` -
+   [special hidden folder](https://developers.google.com/drive/api/v2/appdata)
+   (unique per user) meant to store application-specific data. This is a good
+   choice to prevent accidentally deleting remote storage data from the Google
+   Drive web UI.
+
+```dvc
+$ dvc remote add mygdappata gdrive://appDataFolder
+```
+
+## Configure custom Google Cloud project & app (recommended)
+
+Optionally, follow this guide to setup your own Google Cloud project and
+generate OAuth credentials for your <abbr>DVC projects<abbr> to connect to
+Google Drive. We highly recommend this for heavy and advanced use because:
+
+- you control your Google API usage and rate limits, being able to request an
+  increase to Google if needed.
```

Comment:
to request .. to? do we need a second to? should it be just "request Google to increase"?